spotmarketrequest_test.go

```diff
@@ -45,7 +51,7 @@ func TestAccSpotMarketRequestBasic(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	defer deleteSpotMarketRequest(t, c, smr.ID, true)
+	deleteSpotMarketRequest(t, c, smr.ID, true)
```

Comment:
How will https://github.com/packethost/packngo/pull/220/files#diff-41bd36b03ba2cb5809498cb1c29318a7e8f623d5d76df3c202f4c18aa174a771R73 succeed if the SMR is deleted here?  I think we should continue to `defer` this.

Reply:
ah, you're right. I removed the defer from the deleting function call in TestAccSpotMarketRequestPriceAware, and automatically removed this one too. Thanks for pointing it out.

Reply:
I will fix it, verify that both of the tests run, and then I will push here.

Reply:
@displague fixed. I've just verified that both tests run.